packages/app-insights/src/react/utils.ts

```diff
@@ -0,0 +1 @@
+export const getPrimaryDomain = () => window.location.hostname.split('.').slice(-2).join('.');
```

Comment:
What if the domain is like `foo.com.cn` or `bar.co.jp`...? Should be fine for our own usages, though.

Reply:
discussed offline, add some comment for now